pom.xml

```diff
@@ -2,16 +2,12 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
     <artifactId>virtual-schema-common-document</artifactId>
+    <version>6.1.0-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>6.1.0</version>
```

Reply:
I'm not going  to release now. Some more features are still waiting ;-)